### PR TITLE
[MANOPD-84427] : No busybox version 1.31.2

### PR DESCRIPTION
**Pull request by dmyar21**
### Description
* Need update busybox in globals.yaml. Because there is no previous version on Docker HUB 
* Local-path-provisioner PVC stuck is not working correctly
* Need update json schemas

WA:
* Must be added to `cluster.yaml`. In the `plugins->local-path-provider` section, add the `helper-pod-image` parameter with the value `library/busybox:1.34.1`
* Restart pod local-path-provider

```yaml
plugins:
  local-path-provisioner:
    install: true
    storage-class:
      is-default: 'true'
    helper-pod-image: library/busybox:1.34.1
```

### Solution
* Update busybox in globals.yaml and json schema
### Checklist
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Integration CI passed
- [ ] Unit tests. If Yes list of new/changed tests with brief description
- [ ] There is no merge conflicts

### CODE REVIEW

**Comment by koryaga** [comment]
pls add more info about purpose: bug in previous,  security,  regular update ...

**Review by OleksandrIaresko — APPROVED** [review]

**Comment by OleksandrIaresko** [comment]
@dmyar21 do we need to update version of busybox that mentioned here ?
https://github.com/Netcracker/KubeMarine/blob/main/kubemarine/resources/schemas/definitions/plugins/local-path-provisioner.json#L38

File: kubemarine/resources/schemas/definitions/plugins/local-path-provisioner.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
     },
     "helper-pod-image": {
       "type": "string",
-      "default": "library/busybox:1.35.0"
```
Comment (koryaga) [inline kubemarine/resources/schemas/definitions/plugins/local-path-provisioner.json:38]: I believe no need _library_ prefix for dockerhub

Comment (dmyar21) [reply kubemarine/resources/schemas/definitions/plugins/local-path-provisioner.json:38]: done

Comment (ilia1243) [reply kubemarine/resources/schemas/definitions/plugins/local-path-provisioner.json:38]: Put simply, the simplest solution would be to remove "default" for both "image" and "helper-pod-image"

Comment (koryaga) [reply kubemarine/resources/schemas/definitions/plugins/local-path-provisioner.json:38]: @dmyar21 pls remove

Comment (dmyar21) [reply kubemarine/resources/schemas/definitions/plugins/local-path-provisioner.json:38]: done

**Review by ilia1243 — APPROVED** [review]

**Pull request merged**